types/preview/ember/index.d.ts

```diff
@@ -240,8 +240,6 @@ declare module 'ember' {
     namespace ENV {
       const EXTEND_PROTOTYPES: {
```

Comment:
```suggestion
      const EXTEND_PROTOTYPES: boolean | {
```

Should this be this ^ as well? Since `EXTEND_PROTOTYPES: false` also works.

Reply:
If it *only* allows `false`, then:

```suggestion
      const EXTEND_PROTOTYPES: false | {
```

Otherwise, go for your suggestion!

Reply:
`true` also works indeed.

Reply:
Updated!